test/nbrowser/DropdownConditionEditor.ts

```diff
@@ -31,7 +31,13 @@ describe('DropdownConditionEditor', function () {
         ]);
       });
       await gu.sendKeysSlowly(['hoice not in $']);
+      await gu.sendKeysSlowly(['hoice not in ']);
```

Comment:
Did you mean for this line to replace the previous one (rather than add to it)?

Reply:
Ah, no, good catch! Must have messed up the commit :)